acceptance/tests/pxp-module-puppet/run_puppet_agent_non_root.rb

```diff
@@ -8,7 +8,7 @@
     skip_test "Test is not compatible with #{platform}" if platform =~ /windows/
 
     step 'create non-root user on all nodes' do
-      @user_name = 'foo'
+      @user_name = platform =~ /osx-10.14/ ? 'osx' : 'foo'
```

Comment:
Using the osx user instead of creating a new one for 10.14 due to the new security feature added in this os version that prevents us from changing a user's home directory ( see https://macpaw.com/how-to/full-disk-access-mojave )